spec/jobs/fast_track/disability_compensation_job_spec.rb

```diff
@@ -85,5 +85,28 @@
         end
       end
     end
+
+    context 'when an account for the user is NOT found' do
+      before do
+        allow(Account).to receive(:where).and_return Account.none
+      end
+
+      it 'raises ActiveRecord::RecordNotFound exception' do
+        expect { subject.new.perform(submission.id, user_full_name) }.to raise_error(ActiveRecord::RecordNotFound)
+      end
```

Comment:
Is this spec testing anything in our code? It feels like it's just testing Active Record.

Reply:
I'm more testing that I'm properly raising the error so that we can exit out of the job at that point. Like, if that line was changed again for some reason, I want to make sure we don't forget that.

Reply:
To be specific, the error is only being raised because we're using `.first!` instead of `.first`